spec/researches/passivevoice/getSentencePartsFrenchSpec.js

```diff
@@ -0,0 +1,51 @@
+var getSentenceParts = require( "../../../js/researches/passivevoice/getSentenceParts.js");
+
+describe( "splits French sentences into parts", function() {
+	it ( "returns all sentence parts from the auxiliary to the stopword and from the stopword to the end of the sentence", function() {
+		var sentence = "Ils étaient très amis lorsqu'ils étaient enfants.";
+		expect( getSentenceParts( sentence, "fr" )[ 0 ].getSentencePartText() ).toBe( "étaient très amis" );
+		expect( getSentenceParts( sentence, "fr" )[ 1 ].getSentencePartText() ).toBe( "étaient enfants." );
+		expect( getSentenceParts( sentence, "fr" ).length ).toBe( 2 );
+	} );
+
+	it ( "returns all sentence parts from the auxiliary to the stopword", function() {
+		var sentence = "Ils étaient très heureux lorsque je les y conduisais.";
+		expect( getSentenceParts( sentence, "fr" )[ 0 ].getSentencePartText() ).toBe( "étaient très heureux" );
+		expect( getSentenceParts( sentence, "fr" ).length ).toBe( 1 );
+	} );
+
+	it ( "doesn't split on sentence breakers within words", function() {
+		var sentence = "Désormais tu es responsable de tes actes.";
+		expect( getSentenceParts( sentence, "fr" )[ 0 ].getSentencePartText() ).toBe( "es responsable de tes actes." );
+		expect( getSentenceParts( sentence, "fr" ).length ).toBe( 1 );
+	} );
+
+	it ( "splits sentences on stop characters", function() {
+		var sentence = "C'est en particulier et principalement une question d'argent, a résumé le Premier ministre néerlandais Mark Rutte.";
+		expect( getSentenceParts( sentence, "fr" )[ 0 ].getSentencePartText() ).toBe( "C'est en particulier et principalement une question d'argent" );
+		expect( getSentenceParts( sentence, "fr" ).length ).toBe( 1 );
+	} );
+
+	it ( "doesn't split sentences on stop characters that are not preceded by a word and also not followed by a space/punctuation mark", function() {
```

Comment:
The above test has a stop character preceded by a word AND followed by a space/punctuation mark, and this one has neither.
Can you please add two test to cover the situations where one of those criteria is met?